packages/usa-button-group/src/usa-button-group.twig

```diff
@@ -1,7 +1,47 @@
-<ul class="usa-button-group {{ modifier }}">
-  {% for item in items %}
+<!-- start segmented styles -->
+{% if segmented == true %}
+  <ul class="usa-button-group usa-button-group--segmented">
     <li class="usa-button-group__item">
-        {{ item }}
+      <button class="usa-button" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Map</button>
     </li>
+    <li class="usa-button-group__item">
+      <button class="usa-button usa-button--outline" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Hybrid</button>
+    </li>
+    <li class="usa-button-group__item">
+      <button class="usa-button usa-button--outline" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Satellite</button>
+    </li>
+  </ul>
+
+  {% for modifier in modifiers %}
+  {% if 'usa-button--outline usa-button--inverse' in modifier %}
+    <div class="bg-base-darkest padding-1" style="max-width: fit-content">
+  {% endif %}
+    <ul class="usa-button-group usa-button-group--segmented">
+      <li class="usa-button-group__item">
+        <button class="usa-button {{ modifier }}" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Map</button>
+      </li>
+      <li class="usa-button-group__item">
+        <button class="usa-button {{ modifier }}" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Hybrid</button>
+      </li>
+      <li class="usa-button-group__item">
+        <button class="usa-button {{ modifier }}" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Satellite</button>
+      </li>
+    </ul>
+  {% if 'usa-button--outline usa-button--inverse' in modifier %}
+    </div>
+  {% endif %}
   {% endfor %}
-</ul>
+<!-- end segmented styles -->
+
+<!-- start default styles -->
+  {% else %}
+  <ul class="usa-button-group">
+    <li class="usa-button-group__item">
+      <a href="javascript:void(0);" class="usa-button usa-button--outline" {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Back</a>
+    </li>
+    <li class="usa-button-group__item">
+        <button class="usa-button" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Continue</button>
+    </li>
+  </ul>
```

Comment:
Not sure what this does, but it seems like this template is trying to do three different things. We should clarify where possible.

Reply:
Opened issue https://github.com/uswds/uswds/issues/4881 to resolve in a future enhancement

Reply:
Great, thanks @amyleadem! Resolving this thread.